kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/KyuubiScalaObjectMapper.scala

```diff
@@ -22,8 +22,12 @@ import javax.ws.rs.ext.ContextResolver
 import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
 
-class KyuubiScalaObjectMapper extends ContextResolver[ObjectMapper] {
-  private val mapper = new ObjectMapper().registerModule(DefaultScalaModule)
+import org.apache.kyuubi.server.api.KyuubiScalaObjectMapper.mapper
```

Comment:
ditto, you can import `KyuubiScalaObjectMapper._` in the class

Reply:
Addressed. Thx.